src/components/directorybrowser/directorybrowser.js

```diff
@@ -87,7 +87,7 @@ function getEditorHtml(options, systemInfo) {
     let html = '';
     html += '<div class="formDialogContent scrollY">';
     html += '<div class="dialogContentInner dialog-content-centered" style="padding-top:2em;">';
-    if (!options.pathReadOnly) {
+    if (!options.pathReadOnly && systemInfo.OperatingSystem) {
         const instruction = options.instruction ? `${escapeHtml(options.instruction)}<br/><br/>` : '';
         html += '<div class="infoBanner" style="margin-bottom:1.5em;">';
         html += instruction;
```

Comment:
In theory, `instruction` can still be non-empty.
I think we should check that `instruction` or `systemInfo.OperatingSystem` are not empty.

Reply:
Good catch.